Tests/Marketplace/install_packs.sh

```diff
@@ -19,4 +19,4 @@ echo "starting configure_and_install_packs ..."
 
 python3 ./Tests/Marketplace/configure_and_install_packs.py -s "$SECRET_CONF_PATH" --ami_env "$1" --branch "$CI_COMMIT_BRANCH" --build_number "$CI_PIPELINE_ID" --service_account $GCS_MARKET_KEY -e "$EXTRACT_FOLDER" --cloud_machine "$CLOUD_CHOSEN_MACHINE_ID" --cloud_servers_path $XSIAM_SERVERS_PATH --pack_ids_to_install "$ARTIFACTS_FOLDER/content_packs_to_install.txt" --cloud_servers_api_keys $XSIAM_API_KEYS
 
-exit $RETVAL
+exit $RETVAL
```

Comment:
RETVAL isn't set to any value in this script. it's pointless.
however both the calls to `gcloud auth` and `python ./Test/...` don't check the return value for errors.

Reply:
@daryakoval regarding your [comment](https://github.com/demisto/content/pull/27456#discussion_r1258308874)